pandas/io/excel/_base.py

```diff
@@ -15,7 +15,7 @@
 
 from pandas._libs.parsers import STR_NA_VALUES
 from pandas._typing import Buffer, DtypeArg, FilePathOrBuffer, StorageOptions
-from pandas.compat._optional import import_optional_dependency
+from pandas.compat._optional import _get_version, import_optional_dependency
```

Comment:
we should make this `get_version` and not a private function that is imported across modules. can you make this change as well?